src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,105 @@
-import React from 'react';
 import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
+import { Comment } from '../../react-app-env';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
```

Comment:
I would add "Type" to all types at the end to make it clearer what type it is

And rename this type
```suggestion
type NewCommentFormType = {
```